### PR TITLE
Create RCMAP V5 COVER

### PR DESCRIPTION
**Pull request by ajgriebel**
Create RCMAP V5 COVER

### CODE REVIEW

**Review by simonff — COMMENTED** [review]
Could you change the PR description from "Create V5 COVER" to "Create RCMAP V5 COVER"?

File: catalog/USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+
+local template = import 'RCMAP.libsonnet';
+
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+template + {
+id: id,
```
Comment (simonff) [inline catalog/USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER:14]: please add
 'gee:skip_indexing': true,
 (this is a workaround for the fact we don't yet accept dataset images)

**Review by simonff — COMMENTED** [review]
Could you also add the corresponding JS example in https://github.com/google/earthengine-catalog/tree/main/examples ?

File: catalog/USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER (hunk 1 above)
Comment (ajgriebel) [reply catalog/USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER:14]: That is on line 1. Is that the correct place for it?

Comment (ajgriebel) [reply catalog/USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER:14]: Added as line 15

**Comment by ajgriebel** [comment]
corresponding JS example in https://github.com/google/earthengine-catalog/tree/main/examples 
(Done)

File: catalog/USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER [hunk 2]
```diff
@@ -0,0 +1,24 @@
+'gee:skip_indexing': true,
```
Comment (simonff) [inline catalog/USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER:1]: 'gee:skip_indexing': true should be a part of JSON - compare with https://github.com/google/earthengine-catalog/blob/main/catalog/GOOGLE/GOOGLE_GLOBAL_CCDC_V1.jsonnet#L15

Sorry, I added the comment to line 1 just to make it a general file comment.

File: catalog/USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER (hunk 1 above)
Comment (ajgriebel) [reply catalog/USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER:14]: Removed from line 1, have on line 13

**Comment by ajgriebel** [comment]
// Import the NLCD RCMAP collection.
var dataset = ee.ImageCollection('USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER');

// Filter the collection to the 2019 product.
var nlcd2019 = dataset.filter(ee.Filter.eq('system:index', '2019')).first();

// Each product has multiple bands for different rangeland categories.
print('Bands:', nlcd2019.bandNames());

// Select the rangeland_annual_herbaceous band.
var landcover = nlcd2019.select('rangeland_annual_herbaceous');

var vis = {
  'palette': [
      "000000", "f9e8b7", "f7e3ac", "f0dfa3", "eedf9c", "eada91", "e8d687", "e0d281", "ddd077", "d6cc6d",
      "d3c667", "d0c55e", "cfc555", "c6bd4f", "c4ba46", "bdb83a", "bbb534", "b7b02c", "b0ad1f", "adac17",
      "aaaa0a", "a3a700", "9fa700", "9aa700", "92a700", "8fa700", "87a700", "85a700", "82aa00", "7aaa00",
      "77aa00", "70aa00", "6caa00", "67aa00", "5fa700", "57a700", "52a700", "4fa700", "4aa700", "42a700",
      "3ca700", "37a700", "37a300", "36a000", "369f00", "349d00", "339900", "339900", "2f9200", "2d9100",
      "2d8f00", "2c8a00", "2c8800", "2c8500", "2c8400", "2b8200", "297d00", "297a00", "297900", "277700",
      "247400", "247000", "29700f", "2c6d1c", "2d6d24", "336d2d", "366c39", "376c44", "396a4a", "396a55",
      "3a6a5f", "3a696a", "396774", "3a6782", "39668a", "376292", "34629f", "2f62ac", "2c5fb7", "245ec4",
      "1e5ed0", "115cdd", "005ae0", "0057dd", "0152d6", "0151d0", "014fcc", "014ac4", "0147bd", "0144b8",
      "0142b0", "0141ac", "013da7", "013aa0", "01399d", "013693", "013491", "012f8a", "012d85", "012c82", "01297a",
  ]
};

// Display land cover on the map.
Map.setCenter(-114, 38, 6);
Map.addLayer(landcover, vis, 'Rangeland Annual Herbaceous %');

**Comment by schwehr** [comment]
Closing this pr:

- The file name is incorrect.  Should be USGS/NLCD_RELEASES/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_COVER.jsonnet
- I am going to undo the templating to make changes more explicit.  I will add templating back when all the desired changes are in.

**Pull request closed without merging**